cleaner.py

```diff
@@ -31,79 +28,73 @@ def chunks(l, n):
             yield l[i:i + n]
 
     @staticmethod
-    def get_all_dialogs():
+    def get_all_chats():
         dialogs = app.get_dialogs(pinned_only=True)
 
         dialog_chunk = app.get_dialogs()
         while len(dialog_chunk) > 0:
             dialogs.extend(dialog_chunk)
             dialog_chunk = app.get_dialogs(offset_date=dialogs[-1].top_message.date)
 
-        return dialogs
+        return [d.chat for d in dialogs]
 
-    def select_group(self):
-        dialogs = self.get_all_dialogs()
-        groups = [d for d in dialogs if d.chat.type in ('group', 'supergroup')]
+    def select_groups(self):
+        chats = self.get_all_chats()
+        groups = [c for c in chats if c.type in ('group', 'supergroup')]
 
+        print('Delete all your messages in')
         for i, group in enumerate(groups):
-            print(f'{i+1}. {group.chat.title}')
+            print(f'  {i+1}. {group.title}')
+        print(f'  {len(groups) + 1}. (!) ALL OF THE ABOVE (!)\n')
```

Comment:
Make this option more explicit for user by printing more accurate line please. Also add confirmation step afterwards to prevent casualties.